content/docs/01.getting-started.md

```diff
@@ -31,6 +31,20 @@ docker compose up -d
 
 Open [http://localhost:8080](http://localhost:8080) in your browser.
 
+### Some warnings regarding this docker compose
```

Comment:
```suggestion
### Networking in this docker compose
```

People will skip this part if they don't interesting in networking so the content can be simplified